src/app/components/LayerPicker/LayerDetails.tsx

```diff
@@ -68,6 +68,14 @@ const getDetails = (t: TFunction): Details => ({
       chainDecimalId: '23295',
       docs: docs.sapphire,
     },
+    [Layer.pontusx]: {
+      description: t('layerPicker.testnet.pontusx'),
+      rpcHttp: 'https://pontusx.not.si:443',
+      // rpcWebSockets: 'wss://testnet.sapphire.oasis.dev/ws', // TODO: fix this
```

Comment:
WS support is not even that important. Not sure if Sapphire wrapper even supports it(I could be wrong). The important one is RPC one.

Reply:
Yes. It's not that important, that's why I made that an optional field in this description type, and updated the UI code to handle cases when it's missing. I think we can safely merge with this one field missing.

Reply:
I would simply delete this line.

Reply:
Deleted.